src/controller/users.js

```diff
@@ -1,26 +1,42 @@
 const router = require("express").Router(),
-	updateUser = require("../actions/update-user"),
-	findUser = require("../actions/find-user");
+	auth = require("../middlewares/bearer"),
+	updateUser = require("../actions/users/update-user"),
+	findUser = require("../actions/users/find-user"),
+	notLoggedIn = {
```

Comment:
Yes, it is intentional, it can't be imported like the default sequelize classes can be imported. Haven't found a way to do it decently. If you have an idea, do tell me!